### ISSUE TITLE
z vs zoxide

### ISSUE THREAD

**Issue by Susensio** [jethrokuan/z#113]
How does this `fish` plugin compares to `zoxide`?

**Comment by lydell** [comment]
I’ve been using zoxide for a year or so. Tried this plugin for a couple of minutes. This plugin does not appear to have zoxides’s “also works as a regular cd command” feature. Apparently I’m so used to having `z` and `cd` in one that I quickly went back to zoxide.

**Comment by yujinyuz** [comment]
The only difference I saw was that with this plugin, if you press `z<TAB>` it will show all, while in zoxide it doesn't.
You have to press `z foo<SPACE><TAB>` to get the actual completion.

So, I guess that's just how it works because `z` also acts as `cd`

**Comment by jeeftor** [comment]
I'm using zoxide - but I prefer the z<tab> funcitonality and am thinking of bailing on zoxide due to this annoyance

**Comment by mlncn** [comment]
I am confused— i want to switch to fish from bash so installed this.  I now realize i have been using Zoxide, not — but all that aside, the behavior i am used and want is:

I am in a directory with subdirectories `something` and `other`; if i type `z som<tab>` it will autocomplete to `z something/` and take me there.  And of course if i am later somewhere else entirely, and type `z som<enter>`, it will take me there.

With this plugin and fish however tabs do not autocomplete to the directories i am in.  Fish (or z?) will sometimes suggest random completions that i need to press the left arrow to accept, but they are not generally anything that works with z ("did not match any results").

This `z` does not also act as `cd` for me— `z ../` does not work for example.

In short my experience is what lydell noted, not what yujinyuz noted, and i am wondering if there are different configurations or ways of installing (i did `fisher install jethrokuan/z`) that would make a difference?

**Comment by jeeftor** [comment]
I also used z and found it superior because a single command I could hit tab for completion
